src/helpers/hotkeysManager.js

```diff
@@ -65,7 +64,17 @@ WebViewer(...)
     }
 
     // https://github.com/jaywcjlove/hotkeys#defining-shortcuts
-    hotkeys(key, handler);
+    const { keyup, keydown } = handler;
+    const keyuphandler = keyup || (() => {});
```

Comment:
you can do this instead here:

```js
const {
    keyup = () => {},
    keydown = handler
} = handler
```

Reply:
Also another common practice here is to define a `NOOP` function at the top of the file, like so:
```
const NOOP = () => {}
```